pipeline/steps/tf-init.yaml

```diff
@@ -1,24 +1,23 @@
 parameters:
-  resourceGroup: ''
-  storageAccount: ''
+  subscription: ''
 
 steps:
 
-  - task: TerraformCLI@0
-    displayName: Terraform init
-    inputs:
-      command: init
-      workingDirectory: $(System.DefaultWorkingDirectory)/infrastructure/terraform/
-      backendType: azurerm
-      backendServiceArm: $(subscription)
-      backendAzureRmResourceGroupName: ${{ parameters.resourceGroup }}
-      backendAzureRmResourceGroupLocation: $(location)
-      backendAzureRmStorageAccountName: ${{ parameters.storageAccount }}
-      backendAzureRmContainerName: $(prefix)$(product)terraform
-      backendAzureRmKey: $(prefix)$(product).tfstate
+- task: TerraformCLI@0
+  displayName: Terraform init
+  inputs:
+    command: init
+    workingDirectory: $(System.DefaultWorkingDirectory)/infrastructure/terraform/
+    backendType: azurerm
+    backendServiceArm: ${{ parameters.subscription }}
+    backendAzureRmResourceGroupName: $(devopsResourceGroup)
+    backendAzureRmResourceGroupLocation: $(location)
+    backendAzureRmStorageAccountName: $(devopsStorageAccount)
+    backendAzureRmContainerName: $(prefix)$(product)terraform
+    backendAzureRmKey: $(prefix)$(product).tfstate
```

Comment:
where are prefix and product coming ffrom...pass them as variables

Reply:
Imported from HMI-APIM-Common variable group